release/RELEASE.md

```diff
@@ -55,6 +56,40 @@ patches we backport to the 1.0 release branch).
    where `$VERSION` is the full version tag of the new release you are making,
    i.e. the second column of the `LATEST` file.
 
+   > ## Tips for Windows testing in an ad-hoc machine
```

Comment:
Why is this all in a quote block?

Reply:
My reasoning was that this is an aside. I expect at least one other person than me will have trouble using Windows, just because they don't use it very often, and this is a bit of help for them. But strictly speaking it has nothing to do with testing the release and can be safely ignored. It's a bit of advice, but not a prescription of what must be done for the test to be valid.

That said, happy to remove the quotes.

Reply:
Sounds reasonable, happy to keep the quotes.